tests/unit/cmd/auth/test_login.py

```diff
@@ -437,8 +437,10 @@ def test_invalid_token_exits_error(self, cli_fs_runner, monkeypatch):
     @pytest.mark.parametrize("used_port_count", [0, 1, 10])
     @pytest.mark.parametrize("existing_expired_token", [False, True])
     @pytest.mark.parametrize("existing_unrelated_token", [False, True])
+    @pytest.mark.parametrize("downsized_token", [False, True])
```

Comment:
Add a `None` case where you don't set the field at all in the response's payload to make sure ggshield is compatible with previous versions of the API